static/app/views/organizationStats/index.tsx

```diff
@@ -71,24 +77,32 @@ const UsageStatsOrganization = HookOrDefault({
 });
 
 export class OrganizationStats extends Component<Props> {
-  get dataCategory(): DataCategory {
+  get dataCategory(): DataCategoryInfo['plural'] {
```

Comment:
Anywhere the `DataCategory` type was used is being replaced with `DataCategoryInfo['plural']` to ensure developers know they're using the pluralized string